src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -5872,5 +5872,46 @@ static string F(int a, int b, int c)
     }
 }");
         }
+
+        [WorkItem(62)]
```

Comment:
We totally should! Currently, the WorkItem attribute doesn't take a string. Maybe file an issue? @KevinH-MS was the original implementer of WorkItem. We used it in TFS for tracking unit tests to bugs.